actors/miner/src/lib.rs

```diff
@@ -250,7 +250,7 @@ impl Actor {
 
     /// Returns whether the provided address is "controlling".
     /// The "controlling" addresses are the Owner, the Worker, and all Control Addresses.
-    fn is_controlling_address(
+    pub fn is_controlling_address(
```

Comment:
Please revert, you shouldn't need this when you use inter-actor message sending via the runtime.

Reply:
fixed